src/status_im/mailserver/core.cljs

```diff
@@ -384,7 +397,7 @@
         (fx/merge cofx
                   {:db (dissoc db :mailserver/connection-checks)}
                   (when (= :connecting (:mailserver/state db))
-                    (change-mailserver cofx)))
+                    (change-mailserver)))
```

Comment:
interesting i thought it should be #(when (change %))

Reply:
it can be `#(when (change %))`, literally the same code